### ISSUE TITLE
Feature request: Print auction begin, reveal begin, auction end.

### ISSUE THREAD

**Issue by fortran77** [mattdf/ens-check#3]
It would be very useful if the program and the website printed all three important dates:

  When the auction begins.
  When the reveal period begins.
  When the auction ends.

**Comment by mattdf** [comment]
2nd and 3rd are not possible without using an RPC connection to a geth node, use the registrar app if you want that.

**Comment by fortran77** [comment]
You're right, of course. I had overlooked the fact that the beginning of the reveal period depends on when the auction actually begins, and the auction doesn't begin until somebody makes it begin. So you can't calculate these from the domain itself.

**Comment by skodumur** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
cipherchabon, @mrjohannchang, @yonglid, @a2468834, @maxmetagravity, @kenosan, @RookieCol, @fennar01, @afrideva, @aditya520, @jacobkaufmann, @J2R5M3, @demining, @gkze, @mopig, @kartset, @ronbeltran, @AlbertoCentonze, @girishramnani, @pajamaw
